OrbitCore/CaptureData.h

```diff
@@ -54,7 +54,26 @@ class CaptureData {
     address_infos_ = std::move(address_infos);
   }
 
-  orbit_client_protos::LinuxAddressInfo* GetAddressInfo(uint64_t address);
+  [[nodiscard]] orbit_client_protos::LinuxAddressInfo* GetAddressInfo(
+      uint64_t address);
+
+  [[nodiscard]] const absl::flat_hash_map<int32_t, std::string>&
+  thread_names() {
+    return thread_names_;
+  }
+
+  [[nodiscard]] const std::string& GetThreadName(int32_t thread_id) {
```

Comment:
Can this method be `const`?

Reply:
Made all access methods in CaptureData const where possible.